sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveTableScanSuite.scala

```diff
@@ -166,13 +166,11 @@ class HiveTableScanSuite extends HiveComparisonTest with SQLTestUtils with TestH
              |PARTITION (p1='a',p2='c',p3='c',p4='d',p5='e')
              |SELECT v.id
            """.stripMargin)
-        val plan = sql(
-          s"""
-             |SELECT * FROM $table
-           """.stripMargin).queryExecution.sparkPlan
+        val plan = sql(s"SELECT * FROM $table").queryExecution.sparkPlan
         val relation = plan.collectFirst {
           case p: HiveTableScanExec => p.relation
         }.get
+        // This test case is to verify `hiveQlTable` and `getHiveQlPartitions()`
         val tableCols = relation.hiveQlTable.getCols
```

Comment:
The whole test case for https://github.com/apache/spark/pull/14515 is not needed after the recent code refactoring.